src/cclib/parser/data.py

```diff
@@ -28,6 +28,7 @@ class ccData(object):
         atommasses -- atom masses (array[1], daltons)
         atomnos -- atomic numbers (array[1])
         atomspins -- atomic spin densities (dict of arrays[1])
+        basisname -- name of the standard basis set (string)

```

Comment:
We already have `gbasis` which specifies the basis set numerically. A string is a very vague way of specifying the basis set. We might want this information for convenience, but definitely not as a separate attribute.